core/Pos/Binary/Merkle.hs

```diff
@@ -4,16 +4,16 @@ module Pos.Binary.Merkle () where
 
 import           Universum
 
-import           Pos.Binary.Class   (Bi (..), Raw, label, putField)
+import           Pos.Binary.Class   (Bi (..), Raw)
 import           Pos.Crypto.Hashing (Hash)
 import           Pos.Merkle         (MerkleRoot (..), MerkleTree (..), mkMerkleTree)
 
 -- This instance is both faster and more space-efficient (as confirmed by a
 -- benchmark). Hashing turns out to be faster than decoding extra data.
 instance (Bi a, Bi (Hash Raw)) => Bi (MerkleTree a) where
-    sizeNPut = putField toList
-    get = label "MerkleTree" $ mkMerkleTree <$> get
+  encode = encode . toList
+  decode = mkMerkleTree <$> decode
 
 instance (Bi a, Bi (Hash Raw)) => Bi (MerkleRoot a) where
-    sizeNPut = putField getMerkleRoot
-    get = label "MerkleRoot" $ MerkleRoot <$> get
+  encode = encode . getMerkleRoot
+  decode = MerkleRoot <$> decode
```

Comment:
For imports formatting we use `stylish-haskell` (it's mentioned in style guide, btw). @neongreen can tell you how to enable it only for specific projects.